controllers/cache_handler.go

```diff
@@ -226,7 +227,11 @@ func (h DisruptionTargetWatcherHandler) findNotifiableEvents(eventsToSend map[st
 					case strings.Contains(lowerCasedMessage, "liveness probe"):
 						eventsToSend[chaosv1beta1.EventLivenessProbeChange] = true
 					case strings.Contains(lowerCasedMessage, "readiness probe"):
-						eventsToSend[chaosv1beta1.EventReadinessProbeChange] = true
+						if h.disruption.Status.HasTarget(event.InvolvedObject.Name) {
```

Comment:
Hard to know if it's the same value than the one use before in `HasTarget` :trollface: you tricked me removing your commit, I can't compare!

Reply:
I am sorry 😇 I like to keep a proper git history, but for the next time I will be careful and just add my new commit 🙏

Reply:
Maybe worth adding a comment saying "If the object of the disruption is in the list of targets, it means it has been injected. The readiness probe is failing during the injection" what do you think?